stage1/init/init.go

```diff
@@ -194,6 +194,8 @@ func getArgsEnv(p *Pod, debug bool) ([]string, []string, error) {
 	}
 	args = append(args, nsargs...)
 
+	args = append(args, "--link-journal=try-host")
```

Comment:
I would say we need a different gate: machinedRegister() checks at run-time whether `machined` on the host is recent enough. But here `--link-journal` is a parameter to nspawn in stage1 (not nspawn on the host).

Stage1 can contain systemd-v215 or systemd-v219 at the moment. systemd-v215 has `--link-journal=host` but not `--link-journal=try-host`. So we should check at build-time the version to avoid `--link-journal` on systemd-v215 (or drop support for systemd-v215 if `RKT_STAGE1_USR_FROM=coreos` is updated to use v219 before this PR gets merged).